documentation/modules/overview/con-feature-gates-overview.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// overview/assembly-overview-components.adoc
+
+[id='con-feature-gates-overview-{context}']
+= Feature gates in Strimzi Operators
+
+You can enable and disable some features of operators using _feature gates_. 
+When enabled, a feature gate alters the behavior of the relevant operator.
```

Comment:
That _behavior_ of the relevant operator thing. Unless behavior means something different, maybe we don't need this sentence here as the relevant info is in the previous one.